docs/advanced/storagenetwork/storagenetwork.md

```diff
@@ -0,0 +1,240 @@
+    - `kubectl get -A vmi`
+- All pods that are attached to Longhorn Volumes should be stopped.
+    - Users could skip this step with the Harvester Storage Network setting. Harvester will stop Longhorn-related pods automatically.
+
+## Configuration Example
+
+- VLAN ID
+	- Please check with your network switch setting, and provide a dedicated VLAN ID for Storage Network
+- Well-configured Cluster Network and VLAN Config
+	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
+- IP range for Storage Network
+	- IP range should not conflict or overlap with Kubernetes cluster networks(both `10.42.0.0/16` and `10.43.0.0/16` are reserved).
+	- IP range should be IPv4 CIDR format and 4 times the number of your cluster nodes. Longhorn will use 2 IPs per node, and the upgrade process will run two versions of Longhorn simultaneously. It will consume 4 IPs per node in the upgrade process.
+	- If your cluster has 250 nodes, the IP range should be larger than `/22`.
+
+
+We will take the following configuration as an example to explain the details of the Storage Network
+
+- VLAN ID for Storage Network: `100`
+- Cluster Network: `storage`
+- IP range: `192.168.0.0/24`
+
+## Configuration Process
+
+Harvester will create Multus NetworkAttachmentDefinition from the configuration, stop pods related to Longhorn Volume, update Longhorn setting, and restart previous pods.
+
+### Before Applying Harvester Storage Network Setting
+
+Here we have two cases.
+- Expect that VM VLAN traffic and Longhorn Storage Network use the same group of physical interfaces.
+- Expect that VM VLAN traffic and Longhorn Storage Network use different physical interfaces.
+
+Longhorn will send replication traffic through the specific interfaces shown as the red line in the figure.
+
+#### Same Physical Interfaces
+
+Take `eth2` and `eth3` as an example for VM VLAN traffic and Longhorn Storage Network simultaneously.
+
+Please refer Networking page to configure `ClusterNetwork` and `VLAN Config` with `eth2` and `eth3` and remember the `ClusterNetwork` name for the further step.
+
+![storagenetwork-same.png](./storagenetwork-same.png)
+
+
+#### Different Physical Interfaces
+
+`eth2` and `eth3` are for VM VLAN Traffic. `eth4` and `eth5` are for Longhorn Storage Network.
+
+Please refer Networking page to configure `ClusterNetwork` and `VLAN Config` with `eth4` and `eth5` for Storage Network and remember the `ClusterNetwork` name for the further step.
+
+![storagenetwork-diff.png](./storagenetwork-diff.png)
+
+### Harvester Storage Network Setting
+
+Harvester Storage Network setting will need `range`, `clusterNetwork`, `vlan` field to construct Multus NetworkAttachmentDefinition for Storage Network usage. You could apply this setting via Web UI or CLI.
+
+#### Web UI
+
+Harvester Storage Network setting could be easily modified on the `Settings > storage-network` page.
+
+![storagenetwork-ui.png](./storagenetwork-ui.png)
```

Comment:
Please help re-capture the screenshot with VLAN ID 100 (consistent with the example).

Reply:
fix